tests/granulate_utils/config_feeder/fixtures/dataproc.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import json
+from typing import Any, Dict
+
+from tests.granulate_utils.config_feeder.fixtures.base import NodeMockBase
+
+
+class DataprocNodeMock(NodeMockBase):
+    def __init__(
+        self,
+        *,
+        cluster_uuid: str = "",
+        cluster_name: str = "",
+        instance_id: str = "",
+        is_master: bool = False,
+        region: str = "us-central1",
+        cluster_info: Dict[str, Any] = {},
+    ) -> None:
+        super().__init__()
+
+        metadata = {
+            "id": instance_id,
+            "attributes": {
+                "dataproc-cluster-uuid": cluster_uuid,
+                "dataproc-cluster-name": cluster_name,
+                "dataproc-role": "Master" if is_master else "Worker",
+                "dataproc-region": region,
+            },
+        }
```

Comment:
Please provide source where this example used for tests was taken from

Reply:
added docs: https://github.com/Granulate/granulate-utils/pull/158/files#diff-bd03eeec15796d11276cc6ff76c6560f23885d7fb5692198345e8af5a32a71bcR12